src/hammer-vlsi/tech_test.py

```diff
@@ -11,20 +11,24 @@
 import unittest
 
 from hammer_vlsi import HammerVLSISettings
-from test import HammerToolTestHelpers, DummyTool, HasGetTech
-from typing import Any, Dict, List
+import test
```

Comment:
`test` is an internal file so I'm hesitant to import it this way. Did a wildcard import not suffice?

Reply:
Because they import each other you can not use `from`.

Reply:
So a circular import situation is going on?

Reply:
Yes. This would be solved by #231

Reply:
#318